README.md

```diff
@@ -68,41 +68,81 @@ And run it:
 
 #### Windows
 
-Not here yet ...
+Not yet supported
 
-### Build it yourself  
+## Build from source
 
-To set up your Go environment - look [here](https://golang.org/doc/code.html).
+### Prerequisites
 
-Install Go (Terrahelp is currently built against 1.7.3)
+Install Go (Terrahelp is currently built against 1.13.x).  The following official resources will guide you through your environment setup. 
 
-    mkdir -p "$GOPATH/src/github.com/opencredo/"
-    git clone https://github.com/opencredo/terrahelp.git "$GOPATH/src/github.com/opencredo/terrahelp"
-    cd "$GOPATH/src/github.com/opencredo/terrahelp"
+* [Getting Started](https://golang.org/doc/install)
+* [Go Documentation](https://golang.org/doc)
 
-*Dependencies*
+Clone the Terrahelp repository.
 
-Terrahelp uses [govendor](https://github.com/kardianos/govendor) to manage it's dependencies, and currently also checks them into Git to enable a seamless build experience. However should you wish to change / upgrade any of these, you can get govendor, and then run the appropriate commands (e.g sync, fetch etc)
+```bash
+mkdir -p "$GOPATH/src/github.com/opencredo/"
+git clone https://github.com/opencredo/terrahelp.git "$GOPATH/src/github.com/opencredo/terrahelp"
+cd "$GOPATH/src/github.com/opencredo/terrahelp"
+```
 
-    go get -u github.com/kardianos/govendor
-    govendor sync
+### Dependencies
 
-*Build it*
+Terrahelp uses Go modules to manage it's dependencies.  During Go's transition to switching on modules by default, Terrahelp is setup to buildusing the vendor directory.
+Supportive targets are prvoided to allow the vendor directory to be recreated if required.
 
-    go install
+### Building and Executing
+
+After a build has completed successfully a binary will be built and placed into a local bin directory.  The following commands build and execute terrahelp.
+
+    make build
+    ./bin/terrahelp -v 
     
-*Test it*
+### Testing
     
-    go test -v ./...
+    make test
 
-*Run it:*
+### Installing and Executing
 
-    terrahelp -v 
-    
-*Want to cross compile it?*
+Assuming that your Go environment is setup correctly, an installed binary will be placed in your $GOPATH/bin directory.
+The following commands install and execute Terrahelp.
+
+    make install
+    terrahelp -v
+
```

Comment:
Requirement the workdir/bin to be part of the path, good practise but not standard